commercial/app/model/commercial/books/BookFinder.scala

```diff
@@ -72,6 +72,9 @@ object MagentoService extends ExecutionContexts with Logging {
     )
   }
 
+  private implicit val singleThreadExecutionContext: ExecutionContext =
+    Akka.system.dispatchers.lookup("play.akka.actor.single-thread")
```

Comment:
Hmmm, I'm torn between having a "single thread" context and having a specific "books feed" one.

I can see people misusing a generic one...